ansible_collections/arista/avd/tests/unit/modules/test_configlet_build_config.py

```diff
@@ -51,8 +51,9 @@ def test_get_configlet_invalid_source(self):
         output = get_configlet()
         assert output == dict()
 
-    def test_get_configlet_none_prefix(self):
+    @pytest.mark.parametrize("prefix", ['none', "", None])
+    def test_get_configlet_none_prefix(self, prefix):
```

Comment:
There is another test "test_get_configlet_blank_prefix" which fails now. Since you are now including that scenario here, the other test should be removed. But this also means that the changes are actually breaking in case someone has a blank prefix today. 
 @carlbuchmann can we accept this for minor?